docs/reference/semantic-segmentation.md

```diff
@@ -0,0 +1,157 @@
+## semantic_segmentation module
+
+The *semantic segmentation* module contains the *BisenetLearner* class, which inherit from the abstract class *Learner*.
+
+
+### Class BisenetLearner
+Bases: `engine.learners.Learner`
+
+The *BisenetLearner* class is a wrapper of the BiseNet model [[1]](#bisenetp) found on [BiseNet] (https://github.com/ooooverflow/BiSeNet).
+It is used to train Semantic Segmentation models on RGB images and run inference.
+
+
+
+
```

Comment:
Here there are many empty lines. You should get the same result with just 1 (or 2 empty lines).
```suggestion

```